.editorconfig

```
@@ -0,0 +1,9 @@
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=true
+indent_style=space
+indent_size=4
+
+[{.babelrc,.stylelintrc,.eslintrc,*.json,*.jsb3,*.jsb2,*.bowerrc,*.js}]
+indent_size=2
```

Wollen wir nicht überall lieber 4 nehmen?
```suggestion
indent_size=4
```

---

Im JavaScript durch viel Verschachtelung finde ich 2er wesentlich angenehmer.